### PR TITLE
add some file to gitignore

### PR DESCRIPTION
**Pull request by stewart-yu**
should ignore file generated by `make`  command

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -44,3 +44,11 @@ bazel-*
 # Files generated by JetBrains IDEs, e.g. IntelliJ IDEA
 .idea/
 *.iml
+
+# This is where the result of the go build goes
+/output*/
```
Comment (m1093782566) [inline .gitignore]: There is no `output` directory in our project.

**Comment by m1093782566** [comment]
@stewart-yu 

Please fix the comments.

**Comment by m1093782566** [comment]
@stewart-yu 

Please investigate the test failures. I assume they are not occasionally.

**Comment by stewart-yu** [comment]
Rebase with upstream ,updating code. Waitting ci for moment

**Pull request merged**